app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -92,9 +92,7 @@ table.table-show {
 
 // Following should be set in component library.
 .form-label {
-  margin-bottom: unset;
-  font-size: 1rem;
-  font-weight: 400;
```

Comment:
AFAICT, text with `.form-label` is already sized to 1rem and has a weight of 400.